src/Magnum/Math/Intersection.h

```diff
@@ -428,6 +444,15 @@ template<class T> bool rangeFrustum(const Range3D<T>& range, const Frustum<T>& f
     return true;
 }
 
+template<class T> bool rayRange(const Vector3<T>& rayOrigin, const Vector3<T>& invRayDir, const Range3D<T>& range) {
+    const Vector3<T> t0 = (range.min() - rayOrigin)*invRayDir;
+    const Vector3<T> t1 = (range.max() - rayOrigin)*invRayDir;
+    Vector3<T> tmin, tmax;
+    std::tie(tmin, tmax) = minmax(t0,t1);
```

Comment:
`tie()` unfortunately needs `#include <tuple>` and i banned that one in public headers because it's way too heavy. Access the pair members directly instead, please :)

Also can you rebase on top of current master so CircleCI can build the PR? Thank you.